main.js

```diff
@@ -0,0 +1,68 @@
+var iconDeactivated = document.querySelectorAll(".deactivated");
+var iconActivated = document.querySelectorAll(".activated");
+var categoryInputs = document.querySelectorAll(".category-input");
+var categoryContainer = document.querySelector(".category-container");
+var inputActivity = document.getElementById('activity');
+var inputMinutes = document.getElementById("minutes");
+var inputSeconds = document.getElementById("seconds");
+var buttonStartActivity = document.querySelector(".activity-button");
+var buttonTitles = document.querySelector('.button-title');
+var errorMessages = document.querySelectorAll('.error-message');
+var currentIcon;
+var createdActivities = [];
+
+
+categoryContainer.addEventListener("click", activateIcon);
+buttonStartActivity.addEventListener('click', submitForm);
+
+function toggle(element, className) {
+  element.classList.toggle(className);
+}
+
+function preventDefault(e){
```

Comment:
As of now this function isn't being used, This should be removed.

Reply:
Good catch!